src/server/game/Entities/Object/Object.cpp

```diff
@@ -1089,6 +1089,19 @@ void WorldObject::RemoveFromWorld()
     Object::RemoveFromWorld();
 }
 
+bool WorldObject::IsInWorldPvpZone() const
+{
+    switch (GetZoneId())
+    {
+        case 4197: // Wintergrasp
+            return true;
+            break;
```

Comment:
I wonder if coverity complains about dead code here

Reply:
Wait, why dead code? What am I missing?

Reply:
This would never reach the break statement.

Reply:
:facepalm: 

@jackpoz Only one way to find out, I guess? ;)
I took it straight from [master](https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Entities/Object/Object.cpp#L956). Did anyone do a coverity run on it since Feb. 2017? Because then it should have shown up there.